app/main/views/dashboard.py

```diff
@@ -436,6 +436,50 @@ def get_free_paid_breakdown_for_billable_units(year, free_sms_fragment_limit, bi
         }
 
 
+def format_letter_details_for_month(letter_units_for_month):
+    # Format postage descriptions in letter units e.g. to 'international' not 'europe'
+    for month in letter_units_for_month:
+        for k, v in month.items():
+            if k == 'postage':
+                month[k] = get_postage_description(v)
+
+    # letter_units_for_month must be sorted before international postage values can be aggregated
+    postage_order = {'first class': 0, 'second class': 1, 'international': 2}
+    letter_units_for_month.sort(key=lambda x: (postage_order[x['postage']], x['rate']))
+
+    LetterDetails = namedtuple('LetterDetails', ['billing_units', 'rate', 'cost', 'postage_description'])
+
+    # Aggregate the rows for international letters which have the same rate
+    result = []
+    for _k, g in groupby(letter_units_for_month, key=itemgetter('postage', 'rate')):
+        g = list(g)
+
+        letter_details = LetterDetails(
+            billing_units=sum(x['billing_units'] for x in g),
+            rate=format_letter_rate(g[0]['rate']),
+            cost=(sum(x['billing_units'] for x in g) * g[0]['rate']),
+            postage_description=g[0]['postage']
+        )
+
+        result.append(letter_details)
+
+    return result
+
+
+def get_postage_description(postage):
+    if postage in ('first', 'second'):
+        return f'{postage} class'
+    else:
+        return 'international'
```

Comment:
I think leaving out the `else` makes it clearer that this function returns `international` by default:
```suggestion
    if postage in ('first', 'second'):
        return f'{postage} class'
    return 'international'
```

Reply:
👍 Updated